pycm/pycm_handler.py

```diff
@@ -292,8 +295,10 @@ def __obj_vector_handler__(
         raise pycmVectorError(VECTOR_SIZE_ERROR)
     if len(actual_vector) == 0 or len(predict_vector) == 0:
         raise pycmVectorError(VECTOR_EMPTY_ERROR)
+    if classes is not None and len(set(classes)) != len(classes):
+        raise pycmVectorError(VECTOR_UNIQUE_CLASS_ERROR)
```

Comment:
Isn't it better to throw a warning in this case?

Reply:
@alirezazolanvari 
`classes` order is important so I think the error is better than warning